src/type.ts

```diff
@@ -1,3 +1,99 @@
+export const PERCENTILEGROUPS = [
+  {
+    resourceType: "beds",
+    title: "Hospital beds needed",
+    xAxisLabel: "Dates",
+    yAxisLabel: "Number of beds",
+    charts: [{
+      name: "2.5 percentile",
+      column: "hosp_need_2.5"
+    },
+    {
+      name: "25 percentile",
+      column: "hosp_need_25"
+    },
+    {
+      name: "50 percentile",
+      column: "hosp_need_50"
+    },
+    {
+      name: "75 percentile",
+      column: "hosp_need_75"
+    }],
+  },
+  {
+    resourceType: "ventilators",
+    title: "Ventilators needed",
+    xAxisLabel: "Dates",
+    yAxisLabel: "Number of ventilators",
+    charts: [{
+      name: "2.5 percentile",
+      column: "vent_need_2.5"
+    },
+    {
+      name: "25 percentile",
+      column: "vent_need_25"
+    },
+    {
+      name: "50 percentile",
+      column: "vent_need_50"
+    },
+    {
+      name: "75 percentile",
+      column: "vent_need_75"
+    }],
+  },
+  {
+    resourceType: "icus",
+    title: "ICU's needed",
+    xAxisLabel: "Dates",
+    yAxisLabel: "Number of icu's",
+    charts: [{
+      name: "2.5 percentile",
+      column: "ICU_need_2.5"
+    },
+    {
+      name: "25 percentile",
+      column: "ICU_need_25"
+    },
+    {
+      name: "50 percentile",
+      column: "ICU_need_50"
+    },
+    {
+      name: "75 percentile",
+      column: "ICU_need_75"
+    }],
+  },
+  {
+    resourceType: "deaths",
+    title: "Deaths",
+    xAxisLabel: "Dates",
+    yAxisLabel: "Number of deaths",
+    charts: [{
+      name: "2.5 percentile",
+      column: "death_2.5"
+    },
+    {
+      name: "25 percentile",
+      column: "death_25"
+    },
+    {
+      name: "50 percentile",
+      column: "death_50"
+    },
+    {
+      name: "75 percentile",
+      column: "death_75"
+    }],
+  }
+];
+
+export interface IChartSeriesMetadata {
```

Comment:
Your team might be following a different standard. And in our team there have been debates as well. But generally we don't need to add prefix `I` to interface https://stackoverflow.com/questions/31876947/confused-about-the-interface-and-class-coding-guidelines-for-typescript/41967120#41967120